shared/src/commonMain/kotlin/com.macaosoftware.sdui.app/marketplace/amadeus/auth/login/LoginScreen.kt

```diff
@@ -57,6 +61,19 @@ class LoginScreen(
         val navigator = LocalNavigator.current
         val keyboardController = LocalSoftwareKeyboardController.current
         val coroutineScope = rememberCoroutineScope()
+
+        LaunchedEffect(true) {
```

Comment:
This is not good
`LaunchedEffect(true)`
LaunchedEffect should use a key1.
Also, I don't want to tie the ViewModel logic to the Composable lifecycle, I want to tie it to the Activity lifecycle.

Reply:
It's basically true to launch it whenever it will call for the first time.

Listen, I will show you in the morning and clear your doubts. Because you are not available now.